packages/netlify-cms-proxy-server/src/middlewares/localGit/index.ts

```diff
@@ -231,11 +242,23 @@ export const localGitMiddleware = ({ repoPath, logger }: GitOptions) => {
             const diffs = await getDiffs(git, branch, cmsBranch);
             const label = await git.raw(['config', branchDescription(cmsBranch)]);
             const status = label && labelToStatus(label.trim());
+            const updatedAt =
```

Comment:
Fixes an unrelated bug of dates not showing for editorial workflow entries in local git mode